src/viewer.ts

```diff
@@ -804,15 +802,18 @@ class Viewer {
             });
         };
 
+        // @ts-ignore
+        const maxSamples = device.maxSamples;
```

Comment:
@mvaligursky Should we make this public API?

Reply:
yep I added it to my list. soon.